source/features/first-published-tag-for-merged-pr.tsx

```diff
@@ -15,7 +15,7 @@ const getFirstTag = cache.function(async (commit: string): Promise<string | unde
 		'ul.branches-tag-list li:last-child a'
 	);
 
-	return firstTag?.textContent!;
+	return firstTag?.textContent ?? undefined;
```

Comment:
?

This already meant:

```js
firstTag === null || firstTag === void 0 ? void 0 : firstTag.textContent;
```

`textContent` is never null in practice.

That lengthens it to this, without change in behavior

```js
var _a;
(_a = firstTag === null || firstTag === void 0 ? void 0 : firstTag.textContent) !== null && _a !== void 0 ? _a : undefined;
```

Reply:
It was reported as an error:

> source/features/first-published-tag-for-merged-pr.tsx:18:9
  ✖   18:9  Optional chain expressions can return undefined by design - using a non-null assertion is unsafe and wrong.  @typescript-eslint/no-non-null-asserted-optional-chain

Reply:
Siiiiigh so `@typescript-eslint/no-non-null-asserted-optional-chain` is still broken.

Reply:
I'v disabled that rule for now and reverted this change.